crc-e2e/task.yaml

```
@@ -0,0 +1,325 @@
+    - name: e2e
+      image: quay.io/crcont/crc-e2e:v$(params.crc-version)-$(params.os)-$(params.arch)
+      imagePullPolicy: Always
+      volumeMounts:
+        - name: host-secret
+          mountPath: /opt/host
+        - name: pull-secret
+          mountPath: /opt/pullsecret
+        - name: test-result
+          mountPath: /opt/testresult
+      env:
+      - name: OUTPUT_FOLDER
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)